src/io/builder_proxy.rs

```diff
@@ -33,10 +36,16 @@ impl ColumnBuilderProxy for StringColumnBuilderProxy {
     generic_trait_impl!(VecT::U64);
     fn add(&mut self, string: &str, dictionary: Option<&mut Dict>) -> Result<(), Error> {
         self.commit();
+
+        // TODO: parsing of DSV and Program should be unified
+        // TODO: DSV parsing should depend on logical types
+        let parsed = super::parser::all_input_consumed(parse_dsv_constant)(string.trim())
+            .unwrap_or(string.to_string());
```

Comment:
```suggestion
        let parsed = all_input_consumed(parse_dsv_constant)(string.trim())
            .unwrap_or(string.to_string());
```
Using import instead of fully qualifying the call.